CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 - Fix: [#1763] Title music does not stop when unchecked in options window.
 - Fix: [#1772] Toggling edge scrolling option does not work.
 - Fix: [#1798] Memory leak when resizing the window.
-- Fix: [#1842] Track and Dock objects incorrectly unloaded causing packing issues.
+- Fix: [#1842] Track, Road and Dock objects incorrectly unloaded causing packing issues.
```

Comment:
Original bug?

Reply:
argh true. I'll get it with vehicles I'm sure they have a bug.